types/vfile-rename-tests.ts

```diff
@@ -0,0 +1,20 @@
+import vfile = require('vfile')
+import rename = require('vfile-rename')
+
+const file = vfile('index.js')
+
+function move(file: vfile.VFile) {
+  file.stem = 'main'
+  return file
+}
+
+rename()
+rename('!')
```

Comment:
it would be good to have a return expectation on at least one of the renames

```suggestion
rename('!') // $ExpectType VFile
```